en/docs/guides/login/logging-into-google-using-is.md

```diff
@@ -0,0 +1,170 @@
+
+This page guides you through using WSO2 Identity Server to log in to Google. 
+
+!!! tip "Before you begin!" 
+    You need to have a Google domain. Click
+    [here](https://www.bettercloud.com/monitor/the-academy/create-google-apps-domain-three-easy-steps/)
+    for more information on creating the domain.
+    
+
+## Configure Google
+
+1.  Go to your domain's admin console via
+    `                     https://admin.google.com                   `.
+2.  Click **Security**.
+
+    !!! info 
+		Can't see the Security section? Click the **MORE CONTROLS** bar at
+		the bottom and you can see the Security section.
+
+    ![more-controls](../../assets/img/guides/security-google.png)
+
+3.  Click **Set up single sign-on (SSO) with a third party IdP**.
+
+4.  Enter the following URLs to your third-party Identity Provider
+    (IdP).
+
+    -   **Sign-in page URL:**
+        `             https://<SERVER_HOSTNAME>:9443/samlsso            `
+
+    -   **Sign-out page URL:**
+        `             https://<SERVER_HOSTNAME>:9443/samlsso            `
+
+    ![third-party-idp](../../assets/img/guides/setup-sso-google.png)
+
+5.  Upload the Identity Server certificate:  
+    The certificate file must contain the public key for Google to
+    verify the sign-in requests.
+
+    1.  Navigate to the
+        `            <IS_HOME>/repository/resources/security           `
+        directory via the terminal.
+    2.  Run the command given below to import the public certificate
+        from the keystore to a `            .pem            ` file.
+
+        ``` java
+        keytool -export -alias wso2carbon  -keystore wso2carbon.jks -storepass wso2carbon -file mycert.pem
+        ```
+
+        !!! info 
+			The `              mycert.pem             ` file is created in
+			the same directory mentioned in **step** **a** above. If you
+			want to change the name of the file that is being generated,
+			enter a preferred name instead of
+			`              mycert             ` in the above command.
+
+    3.  Click **Replace certificate** and upload the
+        `           .pem           ` file you just generated.
+
+    ![sso-fill-google.png](../../assets/img/guides/sso-fill-google.png)
+
+    4.  Click **Save**. 
+
+## Configure WSO2 IS to use the email as the username
+
+For details on configuring WSO2 Identity Server to use an email address as the username, refer [Using Email Address as the Username](insert-link).
+
+## Configuring the Service Provider
+
+1.  Sign in to the WSO2 Identity Server [Developer Portal](../../setup/getting-started-with-the-management-console).
+2.  Select **Applications** from the left panel and click on **New Application**.
+3.  Select **SAML web application**. 
+3.  Fill in the **Service Provider Name**, and provide a brief
+    **Description** of the service provider if required. 
+4.  Click **Next**. 
+5.  Fill in the value for **Issuer** and **Assertion consumer URL** as shown below. 
```

Comment:
Won't this be a fragment that we pull?

Reply:
Certain things like the images, assertion, issuer, etc can be different. So I don't think making it a fragment is a good idea.

Reply:
atleast steps 1-4 can be a fragment